pkg/plugins/common/kustomize/v2-alpha/scaffolds/api.go

```diff
@@ -18,6 +18,7 @@ package scaffolds
 
```

Comment:
Hi @Kavinjsir great work, just a few nits:

a) We cannot use 🌱 on this one because this change affects the end-user so that need to be `sparkles` 
b) We need also add this to the migration guide from v3 to v4
- Into Common changes to allow people knows the diff: https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/migration/v3vsv4.md#common-changes
- Into the steps to migrate to allow those that will try to migrate manually knows that they need to create this file and add all kinds there: https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/migration/manually_migration_guide_gov3_to_gov4.md

However, all is great. Just address this both needs it has my lgtm

c/c @everettraven this one helps SDK because their has a plugin manifests that will create this file so that is possible build the bundle with the owned CRDs

Reply:
Hi @camilamacedo86 thx for reminding the documentation. Just added some in the place you suggested.